packages/backend/server/schema.prisma

```diff
@@ -441,6 +441,22 @@ model AiContext {
   @@map("ai_contexts")
 }
 
+model AiWorkspaceConfig {
+  workspaceId String   @map("workspace_id") @db.VarChar
+  // ignored embed docs in the workspace
+  ignoreDocs  String[] @unique @map("ignore_docs") @db.VarChar
```

Comment:
How large will the data volume be for ignoreDocs? Why not create a new association table to do it?

Reply:
since it can only restrict specific docs (server cannot know the associations like collection/tag), it is foreseeable that the data volume will be small and the write frequency will be low
if you are still concerned about performance, I will split file and doc into separate tables later